### ISSUE TITLE
Can't set up secure connection for Zookeeper Nodes

### ISSUE THREAD

**Issue by Tvion** [Altinity/clickhouse-operator#1113]
We are facing the issue, that it's not possible to configure secure connection for Zookeeper nodes by using clickhouse-operator.
```
spec:
  configuration:
    zookeeper:
      nodes:
        - host: <Zookeeper host>
          port: 2181
          secure: 1
```

I found out that "secure" parameter for node is not taken into account and prepared some fix for that.

May you clarify the correct branch for pull request?

**Comment by Slach** [comment]
clickhouse-operator doesn't manage zookeeper itself
Did you activate SSL on zookeeper side? 
look https://cwiki.apache.org/confluence/display/ZOOKEEPER/ZooKeeper+SSL+User+Guide

**Comment by Tvion** [comment]
Yes, Zookeeper is already configured to use secure connection, but we need configure ClickHouse as well for establishing secure connection with Zookeeper.

https://clickhouse.com/docs/en/operations/ssl-zookeeper/

For now it's impossible to apply <secure>1</secure> parameter for Zookeeper node using ClickHouse Operator.

**Comment by Tvion** [comment]
This fix should help
https://github.com/Altinity/clickhouse-operator/pull/1114

**Comment by Slach** [comment]
as quick workaround you could use:

```yaml
spec:
  configuration:
    files:
      config.d/zookeeper_secure.xml: |
        <clickhouse>
          <zookeeper>
            <node>
              <host>your-zookeeper-host</host>
              <port>2281</port>
              <secure>1</secure>
             </node>
          </zookeeper>
        </clickhouse>      
```

**Comment by Tvion** [comment]
Thank you for WA. It works

**Comment by alex-zaitsev** [comment]
Fixed in 0.21.0 by https://github.com/Altinity/clickhouse-operator/pull/1114
